.dotnet/src/Generated/Internal/MultipartFormDataBinaryContent.cs

```diff
@@ -0,0 +1,197 @@
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.ClientModel;
+using System.Globalization;
+using System.IO;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace OpenAI
+{
+    internal class MultipartFormDataBinaryContent : BinaryContent
```

Comment:
I'm not 100% confident this type is correctly implemented, and this implementation removed some Debug.Asserts that seem important.  Is there a way to bring back the custom type for now until we have time to revisit the generator implementation?